app/vulncode_app/views.py

```diff
@@ -1,26 +1,27 @@
-# -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
-from django.http import JsonResponse
-
+from django.http import HttpResponse
+import json
 from django.shortcuts import render
-
 import vulncode_app.api_data as api
 
 def package(request, name):
     """
     Queries the cve-search api with just
     a package name.
     """
-    api.output_cve_id(name=name)
+    raw_data = api.data_cve_circl(name=name)
+    fields_names = ['id', 'summary', 'cvss']
+    extracted_data = api.extract_fields(raw_data, fields_names)
 
-    return JsonResponse({'cve_id':api.ids,'summary':api.summary,'cvss':api.cvss})
+    return HttpResponse(json.dumps(extracted_data))
 
 def package_version(request, name, version):
     """
     Queries the cve-search api with a package
     name and version.
     """
-    api.output_cve_id(name=name, version=version)
+    raw_data = api.data_cve_circl(name=name, version=version)
+    fields_names = ['id', 'summary', 'cvss']
+    extracted_data = api.extract_fields(raw_data, fields_names, version=True)
 
-    return JsonResponse({'cve_id':api.ids,'summary':api.summary,'cvss':api.cvss})
+    return HttpResponse(json.dumps(extracted_data))
```

Comment:
@pombredanne I had changed the data structure of the data, from lists to dictionary and this is sort of what I had implemented.